types/types.go

```diff
@@ -807,7 +807,8 @@ type DiskSpecWithName struct {
 
 // UnmarshalToDisks input format should be:
 // `[{"path":"/mnt/disk1","allowScheduling":false},
-//   {"path":"/mnt/disk2","allowScheduling":false,"storageReserved":1024,"tags":["ssd","fast"]}]`
+//
+//	{"path":"/mnt/disk2","allowScheduling":false,"storageReserved":1024,"tags":["ssd","fast"]}]`
```

Comment:
Why need to change this?

Reply:
My edit changes this by itself....didn't notice it.